src/main/java/com/matera/cadastrocentral/address/AddressController.java

```diff
@@ -0,0 +1,42 @@
+package com.matera.cadastrocentral.address;
+
+import com.sun.org.apache.bcel.internal.generic.NEW;
+import org.apache.catalina.filters.AddDefaultCharsetFilter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/addresses")
```

Comment:
Great job @gdseabra, I think it's good as it is for now, i will create a refactor ticket so we can call the API by "/clients/{clientId}/addresses..." 
I forgot to deescribe it should be client specific, so the blame is on me :facepalm: